sbws/core/generate.py

```diff
@@ -54,8 +54,13 @@ def gen_parser(sub):
     p.add_argument('-m', '--torflow-bw-margin', default=TORFLOW_BW_MARGIN,
                    type=float,
                    help="Cap maximum bw when scaling as Torflow. ")
-    p.add_argument('-r', '--round-digs', '--torflow-round-digs',
```

Comment:
When --torflow-round-digs is passed to sbws, sbws should do what it did in 1.0.

Please revert your changes to lines 57 and 58.